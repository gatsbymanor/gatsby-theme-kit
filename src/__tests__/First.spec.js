import React from 'react';
import renderer from 'react-test-renderer';
import First from '../components/First';

describe('<First />', () => {
  const component = renderer.create(<First />).toJSON();

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});