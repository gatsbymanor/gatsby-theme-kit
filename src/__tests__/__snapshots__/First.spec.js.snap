// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<First /> should render correctly 1`] = `
<section
  className="main special"
  id="first"
>
  <header
    className="major"
  >
    <h2>
      Magna veroeros
    </h2>
  </header>
  <ul
    className="features"
  >
    <li>
      <span
        className="icon major style1 fa-code"
      />
      <h3>
        Ipsum consequat
      </h3>
      <p>
        Sed lorem amet ipsum dolor et amet nullam consequat a feugiat consequat tempus veroeros sed consequat.
      </p>
    </li>
    <li>
      <span
        className="icon major style3 fa-copy"
      />
      <h3>
        Amed sed feugiat
      </h3>
      <p>
        Sed lorem amet ipsum dolor et amet nullam consequat a feugiat consequat tempus veroeros sed consequat.
      </p>
    </li>
    <li>
      <span
        className="icon major style5 fa-diamond"
      />
      <h3>
        Dolor nullam
      </h3>
      <p>
        Sed lorem amet ipsum dolor et amet nullam consequat a feugiat consequat tempus veroeros sed consequat.
      </p>
    </li>
  </ul>
  <footer
    className="major"
  >
    <ul
      className="actions"
    >
      <li>
        <a
          className="button"
          href="generic.html"
        >
          Learn More
        </a>
      </li>
    </ul>
  </footer>
</section>
`;
